sql_upgrade.php

```diff
@@ -295,14 +299,17 @@ function pausePoll(othis) {
                             --$cnt_versions;
                             echo " <option value='$version'";
                             // Defaulting to most recent version or last version in list.
-                            if ($cnt_versions === 0) {
+                            if ($version == ($_POST['form_old_version'] ?? '')) {
+                                echo " selected";
+                            } elseif ($cnt_versions === 0 && !($_POST['form_old_version'] ?? '')) {
                                 echo " selected";
                             }
                             echo ">$version</option>\n";
                         }
                         ?>
                     </select>
-                    <span><?php echo xlt("If you are unsure or were using a development version between two releases, then choose the older of possible releases"); ?>.</span>
+                    <span><?php echo xlt("If you are unsure or were using a development version between two releases, "); ?>
+                    </br> <?php echo xlt("then choose the older of possible releases"); ?>.</span>
```

Comment:
need to keep constants together (no matter how long the line) or else they basically can't be translated

Reply:
could use styling to control the width of the string (ie. force wrapping)